### PR TITLE
Conformance test implementation for Python.

### PR DESCRIPTION
**Pull request by haberman**
This runs the conformance tests under Travis.

### CODE REVIEW

**Comment by haberman** [comment]
Review to @anandolee .

File: conformance/conformance_python.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  elif len(length_bytes) != 4:
+    raise IOError("I/O error")
+
+  # "I" is "unsigned int", so this depends on running on a platform with
+  # 32-bit "unsigned int" type.  The Python struct module unfortunately
+  # has no format specifier for uint32_t.
+  length = struct.unpack("<I", length_bytes)[0]
+  serialized_request = sys.stdin.read(length)
+  if len(serialized_request) != length:
+    raise "I/O error"
```
Comment (anandolee) [inline conformance/conformance_python.py]: IOError("I/O error") ?

File: conformance/conformance_python.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    elif request.WhichOneof('payload') == 'json_payload':
+      try:
+        json_format.Parse(request.json_payload, test_message)
+      except json_format.ParseError as e:
+        response.parse_error = str(e)
+        return response
+
+    else:
+      raise "Request didn't have payload."
```
Comment (anandolee) [inline conformance/conformance_python.py]: Better have exception type

File: conformance/conformance_python.py (hunk 1 above)
Comment (haberman) [inline conformance/conformance_python.py]: Done.

File: conformance/conformance_python.py (hunk 2 above)
Comment (haberman) [inline conformance/conformance_python.py]: Done.

**Comment by anandolee** [comment]
LGTM

**Pull request merged**